Resources/Prototypes/Recipes/Reactions/single_reagent.yml

```diff
@@ -20,6 +20,19 @@
   products:
     EggCooked: 0.5
 
+- type: reaction
+  id: SapBoiling
+  impact: Low
+  minTemp: 377
+  reactants:
+    Sap:
+      amount: 1.2
+  effects:
+  - !type:CreateGas
+    gas: WaterVapor
+  products:
+    Syrup: 0.03 #40:1 ratio based on real syrup production
```

Comment:
Being realistic probably doesn't make for good gameplay- imho make this higher or nobody will bother.

Reply:
Honestly I don't think the nerfs to slowdown, hunger, and such were necessary but w/e